### ISSUE TITLE
Redesign New VFS Team Member gh template

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94448]
## User Story
As a Governance team member, I want to redesign the New VFS Team Member gh template to include only the administrative tasks so that the ticket is simpler for VFS teams to complete and easier for Platform to take action.

Assignee: @allison0034 
Peer Reviewer: @erinrwhite 

## Description
In the last few tickets, Governance has been working to separate the educational tasks associated with Platform Orientation from the administrative tasks. This is so that we can only keep the administrative tasks within the GH template because these are the only tasks Platform actually validates for completion. Platform has our own tasks associated with Platform Orientation, seen under the Platform Checklist section of the current New VFS Team Member gh template, and several of the VFS administrative tasks are prerequisites for the Platform tasks. The educational tasks will soon be moved to live on Platform Website.

The purpose of this ticket is to redesign the New VFS Team Member gh template to limit it only to the administrative tasks for the VFS team member to complete, as well as the Platform Checklist items that Platform will complete. The redesign should also include a link to the educational tasks on Platform Website (this will have to be a placeholder as this page does not yet exist), and the instructions at the top of the current template should be streamlined. 

In December 2022, Governance completed some user research on the Platform Orientation process and the [research findings](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2470543386/Platform+Orientation+Research+Findings) may still be valuable for reference when redesigning the template.

Lastly, it is recommended that the redesign of the New VFS Team Member template be completed in Confluence, and a pull request is NOT created. This is because it still may be sometime before the template is published, and we don't want the pull request to be accidentally merged before it's ready. Do NOT publish any changes to the New VFS Team Member template as part of the scope of this ticket.

### Live pages
- [New VFS Team Member gh template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=platform-orientation%2C+new-vfs-team-member%2C+platform-tech-team-support&projects=&template=orientation-epic.md&title=New+VFS+Team+Member+Template+%5BYour+name+here%5D)

### Drafted pages
- [YAML file](https://github.com/allison0034/testyaml/issues/new?assignees=&labels=&projects=&template=test.yml&title=New+VFS+Team+Member+Template+Your+name+here2)
- [GH PR file](https://github.com/department-of-veterans-affairs/va.gov-team/blob/da42d729769f0e997d6988f8b7dde27bddb98f11/.github/ISSUE_TEMPLATE/orientation-epic.md).

## Tasks
- [x] Using the administrative tasks identified in ticket #94446, create a new draft of the New VFS Team Member template in Confluence
- [x] Ensure the Platform Checklist items remain in the template.
- [x] Ensure we have included a link to the (placeholder) page on Platform Website/Getting Started/Platform Orientation for VFS teams to review the educational tasks
- [x] Try putting yourself in the shoes of a new VFS person. This is the ticket you're told to complete. Does it make sense to you? Is there anything confusing about it? Is there information (in the current state) that you gloss over? How can we make the content instructional without overloading the new team member with too much content? Is it clear who completes which section? Should we consider converting this template into a YAML template?
   - Side note: if we do think a YAML template would be a better user experience for the VFS team member than the current markdown template, you may want to create a test version in your own repo on gh. Please check in with Shira if you are unsure or have questions how to proceed.
- [x] Share your new template design for peer review and feedback

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] We have redesigned the new VFS team member template to no longer include any educational tasks for the VFS team member to complete.
- [x] We have redesigned the New VFS team member template so that it still includes the Platform Checklist items
- [x] We have redesigned the New VFS team member template so that it includes a reference to the educational tasks on Platform Website (note: page does not yet exist so please create as placeholder)
- [x] We have redesigned the New VFS team member template so that the instructions for completing the template are clear, concise and easy to understand and follow

## Team Notification
- [ ] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [x] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by allison0034** [comment]
[Created PR](https://github.com/department-of-veterans-affairs/va.gov-team/pull/96178)

**Comment by shiragoodman** [comment]
@allison0034 are you going to be converting this to a YAML template? Have we considered that? I imagine there's some pros and cons to it. LMK what you think.

**Comment by allison0034** [comment]
@shiragoodman I am not sure yet, I wanted to see what this would look like once restructured.

**Comment by shiragoodman** [comment]
@allison0034 since the template has both VFS actions, and then Platform actions, we'd like to have both there without manually adding anything after the ticket is submitted. IIRC, YAML does not allow for the Platform actions show up after the ticket is submitted... they must be present from the start which I'm not sure will work out too well with VFS teams. I'd imagine they would attempt to complete that section. @briandeconinck does that sound right to you?

I'm wondering if it might make sense to pull Platform Support team in now, since they're the team from Platform who will be interacting with this template, and see what their preference is.

**Comment by allison0034** [comment]
@shiragoodman I just spoke with Brian and I don't think YAML will be a good fit for this. The only benefit would be adding text boxes to the things like "name" but once they submit, they will lose all static text and will not be able to edit unless they look at the markdown file.

**Comment by erinrwhite** [comment]
Reviewed the PR #96178 and left some comments 👍🏻

**Comment by allison0034** [comment]
Chatted with Shira, educational tasks need to be removed. After removing those, I believe this would be a good candidate for YAML... so now [I am YAMLING](https://github.com/allison0034/testyaml/edit/main/.github/ISSUE_TEMPLATE/new-vfs-team-member.yml)!

**Comment by shiragoodman** [comment]
Thank you @allison0034 ! I also think that the Platform Support team should take a look at this when we're done so they can provide their input, since they're the team that will be handling these tickets for Platform. I think the best approach would be to handle that in a separate ticket for the next sprint. I will write it up. Please lmk if you have any questions!

**Comment by allison0034** [comment]
This will move into the next sprint, first time working with YAML so I had a bit of a learning curve.

**Comment by allison0034** [comment]
@shiragoodman I know Erin is out, if you have time do you want to take a [peek at this template](https://github.com/allison0034/testyaml/issues/new?assignees=&labels=&projects=&template=test.yml&title=New+VFS+Team+Member+Template+Your+name+here2)?

**Comment by shiragoodman** [comment]
I took a quick peek. I might have more feedback later, but my big concern now is that we make it VERY clear that the VFS team member is not supposed to complete any items on the Platform Checklist. Maybe we can use a line divide? Or make the Platform Checklist section a higher level heading? 

Also "Tasks for offboarding teammates" is a VFS task, not Platform. We may want to rework the language for that one too. It's supposed to just be informational so that if/when someone on their team leaves, they're aware of the steps they need to take. Maybe it belongs in educational tasks? up for discussion.

**Comment by allison0034** [comment]
@shiragoodman I added some horizontal rules to help break up the sections.

Quick question, off boarding would happen after this ticket was submitted and a team member would have been here for awhile (ideally) correct? For example, Allison starts in October of 2020, completed her tasks in this ticket, clicks create. Then platform completes the tasks in platform. Years later Allison hits the lottery and resigns... VFS opens Allison's ticket and completes the off boarding task. 

Is that the correct flow for off boarding : )

**Comment by allison0034** [comment]
@erinrwhite if you want to review you can, it may change slightly.

**Comment by shiragoodman** [comment]
@allison0034 can you elaborate what you mean when you say, "VFS open's Allison's ticket and completes the off boarding task" ? VFS team (either Allison or someone on behalf of Allison) would open the Offboarding ticket if/when Allison leaves. 

The task related to offboarding in the New VFS Team Member ticket is to _inform_ individuals of the offboarding process, not to execute the offboarding process. The goal of that task is to share information with the VFS so that in the future, if/when they or someone else from their team departs, they're familiar with the process and what's required of them.

**Comment by allison0034** [comment]
@erinrwhite please review the [YAML file](https://github.com/allison0034/testyaml/issues/new?assignees=&labels=&projects=&template=test.yml&title=New+VFS+Team+Member+Template+Your+name+here2) and [GH PR file](https://github.com/department-of-veterans-affairs/va.gov-team/blob/da42d729769f0e997d6988f8b7dde27bddb98f11/.github/ISSUE_TEMPLATE/orientation-epic.md).

**Comment by erinrwhite** [comment]
@allison0034 The YAML looks good! Couple of things

**YAML file name and ticket title** Both have extra 2's in them - should be removed from the ticket metadata

**Link update:** `Getting help from the Platform in Slack` link is broken, should go to https://depo-platform-documentation.scrollhelp.site/support/getting-help-from-the-platform-in-slack

**Formatting for "Tasks for..." headings** - is there a way to make these sub-headings (Tasks for everyone, Tasks for Front-end and Back-end Engineers, etc) h2's or h3's rather than labels? That's more semantically correct here, I think.

**Implementation note: ** It looks like, when we are ready to go live, we will need to remove the `orientation-epic.md` file entirely and replace it with a new `new-vfs-team-member.yml` file using the code that you used in your personal repo. Then we would need to update the link in the [guidance for new team members page](https://depo-platform-documentation.scrollhelp.site/getting-started/guidance-for-new-teams-and-team-members) on platform website to link to the new issue template URL. That work is not part of this ticket, though - just a note for the future.

**Comment by shiragoodman** [comment]
@allison0034 yes, just add as a comment to this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/94452 It still needs refinement, but a comment should suffice for now.

**Comment by erinrwhite** [comment]
@allison0034 

> these are labels on the checkboxes. We could add headings but then what would I name the checkbox labels? Any ideas?

Dah! You're right - I was misunderstanding how this worked. All good.

Two more things I saw:

(1) we need to make sure to add the labels to the new YAML template

`labels: ["platform-orientation", "new-vfs-team-member", "platform-tech-team-support"]`

(2) for the educational tasks, do we want to have it as a checklist item (implying that they should finish those tasks before submitting the ticket) or should we have it as a separate section on the form "After you submit this ticket"?

**Comment by allison0034** [comment]
@erinrwhite yes on the labels! I will add that to the implementation ticket, we are not publishing this yet and I did not want to submit any tickets with those labels right now since we are testing.

I don't think we want them to do the educational tasks after they submit this ticket. I am not even sure if those are going to be check boxes. Since we do not know exactly what that page looks like we will reevaluate once it is complete.

**Comment by erinrwhite** [comment]
Sounds good to me. Review complete and boxes checked! Thanks for the back n forth.

**Comment by shiragoodman** [comment]
@allison0034 In the YAML template, I see the language, "Complete the form and tasks then click create." However there is no "create" button. The button is called "Submit new issue"

This content is also a little funky: "Welcome to the Department of Veterans Affairs (VA) Veteran Facing Services (VFS) team." It makes it sound like they're on a new team called VFS, but that's not accurate. They're on a team that is classified as a VFS. Can we rework the language a bit so it's more clear? Let me know if you have questions about this. 

I think this language may also be confusing:
"Note: this ticket should not be submitted/created until after you receive access to the appropriate repos."
That's not exactly true. I would reach out to Alyssa for her teams input here. Or possibly Andrea if Alyssa can't help. The VFS team member isn't waiting on receiving access to the appropriate repos (that is the step Platform Support completes when the ticket is submitted), but rather they're waiting until they've been adjudicated and fingerprinted by VA. I'm not sure what appropriate language to use, but "appropriate repos" is NOT CORRECT.

**Comment by shiragoodman** [comment]
same comments for the MD file too - thank you!

**Comment by allison0034** [comment]
@shiragoodman I updated the verbiage for the button in the YAML file, in the MD file it is "Create".

Honestly, do we even need a "welcome" message? They get to this form from a web page that explains things. I removed it. Let me know what you think. 

I will reach out to Alyssa about the note.

**Comment by allison0034** [comment]
@shiragoodman @erinrwhite am I good to close?

**Comment by shiragoodman** [comment]
@allison0034 yes, fine by me.

**Comment by erinrwhite** [comment]
@allison0034 all set.
